easybuild/easyconfigs/p/PGI/PGI-16.1-GCC-4.9.2-2.25.eb

```diff
@@ -0,0 +1,26 @@
+name = 'PGI'
+version = '16.1'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgicdk-%(version_major)s%(version_minor)s.tar.gz']
```

Comment:
this is different from all of the other PGI easyconfigs, where we have instead:

```
sources = ['pgilinux-20%(version_major)s-%(version_major)s%(version_minor)s-x86_64.tar.gz']
```

Is this a particular variant of PGI? Do we need a versionsuffix?